modules/notification/action/action.go

```diff
@@ -275,6 +275,26 @@ func (*actionNotifier) NotifyMergePullRequest(pr *models.PullRequest, doer *mode
 	}
 }
 
+func (*actionNotifier) NotifyPullRevieweDismiss(doer *models.User, review *models.Review, comment *models.Comment) {
+	reviewerName := review.Reviewer.Name
+	if len(review.OriginalAuthor) > 0 {
+		reviewerName = review.OriginalAuthor
+	}
+	if err := models.NotifyWatchers(&models.Action{
+		ActUserID: doer.ID,
+		ActUser:   doer,
+		OpType:    models.ActionPullReviewDismissed,
+		Content:   fmt.Sprintf("%d|%s|%s", review.Issue.Index, reviewerName, comment.Content),
```

Comment:
if the point is that the index, reviewername and comment content can be extracted from this - because we're using potentially the original author name here - any `|` in the reviewerName need to be escaped somehow.

Reply:
No, can't get these message (It's an special type comment) by issue index. Then all message used in action feed should be saved because they maybe be changed or removed later. action content just show the message at that time, should not be changed or broken because of these reason.

Reply:
No, can't get these message (It's an special type comment) by issue index. Then all message used in action feed should be saved because they maybe be changed or removed later. action content just show the message at that time, should not be changed or broken because of these reason.